script.js

```diff
@@ -8,6 +8,91 @@ $(document).ready(function () {
     $(".search-options").removeClass("hide");
   });
 
+
+  // function to load last 5 searches in local storage
+  let historyButton1 = $("#button1")
```

Comment:
Suggestion to code: rather than create individual buttons, build a function that will update user search history section automatically
-> create an array ```let userSearches = [];``` 
-> add array to local storage
-> add new user search to the top of the list/array
-> loop that creates buttons and append to html section
-> make sure you clear user search history before it can display new searches 

https://github.com/EAmrogowicz/weather-dashboard/blob/8d51d3bdf477c215f0eeb5378e4509da8289a306/script.js#L23